cli/cmdlineparser.cpp

```diff
@@ -1283,6 +1283,7 @@ void CmdLineParser::printHelp()
     "    --rule-file=<file>   Use given rule file. For more information, see:\n"
     "                         http://sourceforge.net/projects/cppcheck/files/Articles/\n"
 #endif
+    // TODO: document --showtime
```

Comment:
Will add after it is well defined after all my other comments/TODOs have been resolved.